tests/smb.test.ts

```diff
@@ -0,0 +1,9 @@
+import { test, expect } from "bun:test"
+import { convertCircuitJsonToPcbSvg } from "circuit-to-svg"
+import { fp } from "../src/footprinter"
+
+test("smb", () => {
+  const soup = fp.string("smb").circuitJson()
```

Comment:
```suggestion
  const circuitJson = fp.string("smb").circuitJson()
```

This is the 100th time BTW! pointing the same thing 🫡